scm-info/org/apache/geronimo/specs/_artifact/geronimo-jms_2.0_spec/scm.yaml

```diff
@@ -0,0 +1,6 @@
+---
+type: "git"
+uri: "https://github.com/apache/geronimo-specs.git"
+tagMapping:
+  - pattern: (.*)
```

Comment:
This will map every version to that one tag. I don't think you want this.

Reply:
Removed.